### PR TITLE
Ec dh transform api

### PR DESCRIPTION
**Pull request by ilagordhan**
Documents transform api release.

### CODE REVIEW

File: content/apidocs-mxsdk/apidocs/data-hub-apis.md [hunk 1]
```diff
@@ -1,78 +1,113 @@
...
 ---
 
 ## 1. Introduction
 
-The Data Hub API enables you to explore and discover data sources in Mendix Data Hub and use them to build new apps. You can also set up a registration flow in your deployment pipeline to register new data sources from your applications to the Data Hub Catalog.
+The [Data Hub APIs](#datahubapis) can be used to register your data sources to the organization's Mendix Data Hub. You can also explore and discover data sources that can be used in your app development.
+
+Using the APIs you can set up a registration flow in the deployment pipeline of your business applications to register new data sources from your applications to the Data Hub Catalog.
+
+For Mendix users deploying their apps to a non-Mendix environment the [Data Hub Transform API](#transform) is available to generate the request bodies required to register your data sources for you Menidx apps.
```
Comment (NammNguyen) [inline content/apidocs-mxsdk/apidocs/data-hub-apis.md:15]: Should be 'for your Mendix apps' at the end

File: content/apidocs-mxsdk/apidocs/data-hub-apis.md [hunk 2]
```diff
@@ -1,78 +1,113 @@
...
+
+{{% alert type="info" %}}
+For the current release, the interactive features of the OpenAPI interface are not operational and therefore the **Try it out** feature does not work.
+{{% /alert %}}
+
+## 5. Data Hub Transform Open API Spec {#transform}
+
+For Mendix apps deploying to the Mendix cloud, there is a registration pipeline that registers the published OData contracts (data sources), and also the consumed entities (datasets) to Data Hub. Information that is defined in the location constants of the consumed and published services—located in the `dependencies.json` file for the app—is used.
+
+The [Transform API](https://datahub-spec.s3.eu-central-1.amazonaws.com/transform.html) is available for Mendix users who do not deploy to the Mendix cloud. It extracts the information from the `dependencies.json` file and returns a response that can be used in the request bodies for the PUT published endpoints and PUT consumed endpoints calls to the Data Hub API. Information this is not returned by the Transform API and are required are included in []()
```
Comment (NammNguyen) [inline content/apidocs-mxsdk/apidocs/data-hub-apis.md:93]: The last sentence looks incomplete to me?

File: content/apidocs-mxsdk/apidocs/data-hub-apis.md [hunk 3]
```diff
@@ -1,78 +1,113 @@
...
+
+The Transform API is available at: https://datahub-spec.s3.eu-central-1.amazonaws.com/transform.html.
+
+{{% alert type="info" %}}
+For the current release, the interactive features of the OpenAPI interface are not operational and therefore the **Try it out** feature does not work.
+{{% /alert %}}
+
+### 5.2 Location of the `dependencies.json` file of an App
+
+For a Mendix app, the `dependence.json` file is usually located in the project folder of the app under the following directory: `Mendix\<YourApplicationName>\deployment\model`
```
Comment (NammNguyen) [inline content/apidocs-mxsdk/apidocs/data-hub-apis.md:105]: Should be dependencies.json

File: content/apidocs-mxsdk/apidocs/data-hub-apis.md [hunk 4]
```diff
@@ -1,78 +1,113 @@
...
+
+### 5.3 Optional Values not Obtained from `dependencies.json`
+
+The request bodies returned by the Transform API do not contain the values for the following attributes because they’re not available from the `dependencies.json`. When registering endpoints they must still be specified:
```
Comment (NammNguyen) [inline content/apidocs-mxsdk/apidocs/data-hub-apis.md:109]: These are optional fields for the registration APIs so I would use different wording here, registration will still succeed (using default values) if they're left empty

File: content/apidocs-mxsdk/apidocs/data-hub-apis.md (hunk 2 above)
Comment (ilagordhan) [reply content/apidocs-mxsdk/apidocs/data-hub-apis.md:93]: yes - the reference to that section... thanks for reminding me about it. It is all a bit "last minute"!

File: content/apidocs-mxsdk/apidocs/data-hub-apis.md (hunk 4 above)
Comment (ilagordhan) [reply content/apidocs-mxsdk/apidocs/data-hub-apis.md:109]: Yes, I am oscillating between "you can" "if you want to", and in this iteration, thought, you have to (if you want to...) :-)))

File: content/apidocs-mxsdk/apidocs/data-hub-apis.md (hunk 1 above)
Comment (ilagordhan) [reply content/apidocs-mxsdk/apidocs/data-hub-apis.md]: Many thanks

**Pull request merged**